test/train-sets/ref/topk-rec.stderr

```diff
@@ -12,20 +12,17 @@ loss     last          counter         weight    label  predict features
 0.000003 0.000003            1            1.0   3.0000   2.9983        4
 0.000004 0.000005            2            2.0   0.0000   0.0022        4
 0.000003 0.000000            3            3.0   2.0000   2.0000        4
-0.000003     n.a.            4            4.0  unknown   0.0000        1
-0.000003 0.000003            5            5.0   0.0000   0.0018        4
-0.000004 0.000011            6            6.0   3.0000   2.9968        4
-0.000004 0.000001            7            7.0   1.0000   1.0007        4
-0.000004     n.a.            8            8.0  unknown   0.0000        1
-0.000003 0.000000            9            9.0   2.0000   2.0004        4
-0.000003 0.000000           10           10.0   1.0000   1.0003        4
-0.000002 0.000000           11           11.0   3.0000   2.9995        4
-0.000002     n.a.           12           12.0  unknown   0.0000        1
+0.000003 0.000003            4            4.0   0.0000   0.0018        4
+0.000004 0.000011            5            5.0   3.0000   2.9968        4
+0.000004 0.000001            6            6.0   1.0000   1.0007        4
+0.000003 0.000000            7            7.0   2.0000   2.0004        4
+0.000003 0.000000            8            8.0   1.0000   1.0003        4
+0.000002 0.000000            9            9.0   3.0000   2.9995        4
```

Comment:
The number of multiline examples should be 3, right?

Reply:
Yes it is 3, but I am recreating the previous behavior of printing statistics after each example. See line 91 in the reduction. What do you think is the correct thing to do here? Only print the statistics after each set of examples?

Reply:
Yes.   All predictions from a top-k call should (conceptually) be on a single line.  The idea of this reduction is that you are return the top-k-of-n things.